File: php/testcode/benchmark_test_00415.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00415(BenchmarkRequest $req): BenchmarkResponse {
    $data = $req->post('data');
    $digest = hash('sha3-256', $data);
    return BenchmarkResponse::json(['hash' => $digest]);
}

File: rust/testcode/benchmark_test_00415.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let offset: usize = req.param("offset").parse().unwrap_or(0);
    let length: usize = req.param("length").parse().unwrap_or(0);
    let end_index = offset + length;
    super::shared::BenchmarkResponse::ok(&format!("end={}", end_index))
}
